autogpt_platform/frontend/src/components/agents/agent-runs-selector-list.tsx

```diff
@@ -49,15 +49,24 @@ export default function AgentRunsSelectorList({
   onSelectPreset,
   onSelectSchedule,
   onSelectDraftNewRun,
-  onDeleteRun,
-  onDeletePreset,
-  onDeleteSchedule,
+  doDeleteRun,
+  doDeletePreset,
+  doDeleteSchedule,
   className,
 }: AgentRunsSelectorListProps): React.ReactElement {
   const [activeListTab, setActiveListTab] = useState<"runs" | "scheduled">(
     "runs",
   );
 
+  useEffect(() => {
+    // Reset the active tab when the selected view changes
+    if (selectedView.type === "schedule") {
+      setActiveListTab("scheduled");
+    } else {
+      setActiveListTab("runs");
+    }
+  }, [selectedView]);
```

Comment:
```suggestion
  useEffect(() => {
    if (selectedView.type === "schedule") {
      setActiveListTab("scheduled");
    } else {
      setActiveListTab("runs");
    }
  }, [selectedView]);
```
I think no need for this comment given the code is obvious enough in this case?